### PR TITLE
#163360804 users should be able to Update/Delete an article

### PR DESCRIPTION
**Pull request by jillo-abdullahi**
**What does this PR do?**
-  Implements user ability to update and delete articles

**Description of tasks to be completed?**
- Authenticated users can edit their own article
- Authenticated users who are article owners can choose to delete their article(s)

**How should this be manually tested?**

- Clone the repo and cd into the project folder
- Switch to the appropriate branch with `ft-update-delete-article-163360804`
- Run `npm install` to update your packages.
- Run tests with `npm test`
- Fire up the server with `npm start`
- Navigate to `http://localhost:3000/create_article` to test on Postman.

**What are the relevant pivotal tracker stories?**
- [#163360804](https://www.pivotaltracker.com/story/show/163360804)

**Checklist (edited)**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by justmesam** [comment]
I noticed this has work that is in the other PR, we should wait for that to be merged or it will start having merge conflicts.

File: src/assets/style/articles/base/_base.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+html{
+    font-size: 62.5%;
+}
+body {
+    font-family: Helvetica, Arial, sans-serif;
+    font-size: $m-size;
+}
+button {
```
Comment (bettblake08) [inline src/assets/style/articles/base/_base.scss]: Change the button css to sass structure.

File: src/assets/style/articles/base/_settings.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+$purple: #8357c5;
+$blue: #3c4251;
+$light-blue: #464b5e;
+
+
+// Spacing
+$s-size: 1.2rem;
+$m-size: 1.6rem;
+$l-size: 3.2rem;
+$xl-size: 4.8rem;
```
Comment (bettblake08) [inline src/assets/style/articles/base/_settings.scss]: Add a line

File: src/components/views/articles/SingleArticle.js [hunk 3]
```diff
@@ -0,0 +1,235 @@
+/* eslint-disable camelcase */
```
Comment (bettblake08) [inline src/components/views/articles/SingleArticle.js]: Please place the exceptions inline with the code you are trying to exempt.

**Review by bettblake08 — CHANGES_REQUESTED** [review]

File: src/components/views/articles/SingleArticle.js (hunk 3 above)
Comment (jillo-abdullahi) [reply src/components/views/articles/SingleArticle.js]: This has been done. Thanks @bettblake08

File: src/assets/style/articles/base/_settings.scss (hunk 2 above)
Comment (jillo-abdullahi) [reply src/assets/style/articles/base/_settings.scss]: On it!

File: src/assets/style/articles/base/_base.scss (hunk 1 above)
Comment (jillo-abdullahi) [reply src/assets/style/articles/base/_base.scss]: On it!

**Review by Darius-Ndubi — APPROVED** [review]
👍

**Review by bettblake08 — APPROVED** [review]
👍

**Review by Bryan-Cee — APPROVED** [review]
👍 LGTM

**Review by denomash — APPROVED** [review]
👍

**Review by davishooly — APPROVED** [review]

**Comment by justmesam** [comment]
LGTM

**Pull request merged**